examples/e12_global_data/src/main.rs

```diff
@@ -89,10 +89,18 @@ impl EventHandler for Handler {
                 data_read.get::<MessageCount>().expect("Expected MessageCount in TypeMap.").clone()
             };
 
+            // To check how many "owo" there are in msg.content, we split the string with the "owo" pattern 
+            // and reduce the count of iterator returned by the split function by one.
+            let owo_in_msg = msg.content.to_lowercase().split("owo").count() - 1;
+            
             // Atomic operations with ordering do not require mut to be modified.
             // In this case, we want to increase the message count by 1.
             // https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.fetch_add
-            count.fetch_add(1, Ordering::SeqCst);
+            if owo_in_msg == 0 {
```

Comment:
Why this special case? Can it even happen?

Reply:
Yes. If you just send "owo", there will be just one count, because when split try to split it between before "owo" and after "owo",  it would return just one iter, and 1 - 1 = 0. I thought about adding a string " " in the msg content to when a message contet be just "owo" it be splitted to "owo" and " ", did you get it?

Reply:
In the case you describe owo_in_msg = 1
![Screenshot_20220620_221519](https://user-images.githubusercontent.com/21220820/174672277-e8a16642-5861-45ff-bd2f-9ad92bc45477.png)

Reply:
Yes, that's true, i could do it. But check the new commit.